config/crd/bases/ingressnodefirewall.openshift.io_ingressnodefirewallconfigs.yaml

```diff
@@ -96,8 +96,8 @@ spec:
                   description: "Condition contains details for one aspect of the current
                     state of this API Resource. --- This struct is intended for direct
                     use as an array at the field path .status.conditions.  For example,
-                    type FooStatus struct{ // Represents the observations of a foo's
-                    current state. // Known .status.conditions.type are: \"Available\",
+                    \n type FooStatus struct{ // Represents the observations of a
+                    foo's current state. // Known .status.conditions.type are: \"Available\",
```

Comment:
don't know why this is generated now with make manifest maybe something to do with k8s 1.25 update don't know